app/gui/view/welcome-screen/src/template_cards.rs

```diff
@@ -0,0 +1,153 @@
+//! Template cards for Welcome Screen.
+//!
+//! Template cards allows user to select a predefined template by clicking on the corresponding
+//! card.
+
+use ensogl::prelude::*;
+
+use crate::ClickClosure;
+use crate::Frp;
+
+use ensogl::system::web;
+use ensogl::system::web::AttributeSetter;
+use ensogl::system::web::NodeInserter;
+use web_sys::Element;
+use web_sys::HtmlDivElement;
+
+
+
+// ========================
+// === Cards Definition ===
+// ========================
+
+struct Card<'a> {
+    /// Id of HTML element.
+    id:      &'a str,
+    /// Class of HTML element.
+    class:   &'a str,
+    /// URL to background image of the card.
+    img:     Option<&'a str>,
+    /// Header text of the card.
+    header:  &'a str,
+    /// Content text of the card.
+    content: &'a str,
```

Comment:
most of the comments are not needed. We prefer NOT to comment fields in favour of creating descriptive names. For example, `Card::header` - it is aobivous that it is "a header of a card". However the `img` name is bad, because it does not explain that it is background image.

Reply:
> We prefer NOT to comment fields in favour of creating descriptive names.

Noted, it's not mentioned anywhere in contributing and style guides as far as concerned.
Fixed, docs removed, `img` renamed to `image_url`.

Reply:
🤔 actually it should be `background_image_url`, will change it

Reply:
`Noted, it's not mentioned anywhere in contributing and style guides as far as concerned.` - good point – would you be so nice and add it to the guide? Lets add there things on demand and over the time we will refactor it to a new, better state :)